lib/go/test/flow_idtable_staking_test.go

```diff
@@ -2303,3 +2303,246 @@ func TestIDTableStaking(t *testing.T) {
 
 	})
 }
+
+func TestIDTableRewardsWitholding(t *testing.T) {
+
+	t.Parallel()
+
+	b := newBlockchain()
+
+	env := templates.Environment{
+		FungibleTokenAddress: emulatorFTAddress,
+		FlowTokenAddress:     emulatorFlowTokenAddress,
+	}
+
+	accountKeys := test.AccountKeyGenerator()
+
+	var totalPayout interpreter.UFix64Value = 125000000000000 // 1.25M
+	var cutPercentage interpreter.UFix64Value = 8000000       // 08.0 %
+
+	// Create new keys for the ID table account
+	IDTableAccountKey, IDTableSigner := accountKeys.NewWithSigner()
+	idTableAddress, feesAddr := deployStakingContract(t, b, IDTableAccountKey, IDTableSigner, env, true)
+
+	env.IDTableAddress = idTableAddress.Hex()
+	env.FlowFeesAddress = feesAddr.Hex()
+
+	var totalStaked interpreter.UFix64Value = 510000000000000
```

Comment:
a comment here for what this value is (like above) would be great, since `UFix64` values are offset

Reply:
moved it to near the end of the test and made it a calculation